toolchain/parse/node_kind.def

```diff
@@ -261,10 +266,12 @@ CARBON_PARSE_NODE_KIND_CHILD_COUNT(ContinueStatement, 1,
 
 // `return`:
 //   ReturnStatementStart
+//   _optional_ ReturnVarSpecifier
```

Comment:
I think this should probably be something like "ReturnVarSpecifier or expression", not implying both are allowed.

Reply:
Done.